modules/arm_plugin/armplg_build.sh

```diff
@@ -109,15 +110,13 @@ cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DBUILD_LIST=imgcodecs,videoio,highgui,gapi
       -D OPENCV_OTHER_INSTALL_PATH=etc \
       -D OPENCV_LICENSES_INSTALL_PATH=etc/licenses \
       -D ENABLE_CXX11=ON \
-      -D CMAKE_INSTALL_PREFIX=install \
       -DCMAKE_TOOLCHAIN_FILE="$OPENVINO_HOME/cmake/$TOOLCHAIN_DEFS" \
-      -DCMAKE_STAGING_PREFIX=$STAGING_DIR/opencv \
       -DWITH_GTK_2_X=OFF \
       -DOPENCV_ENABLE_PKG_CONFIG=ON \
       -DPKG_CONFIG_EXECUTABLE=/usr/bin/${ARCH_NAME}-pkg-config \
```

Comment:
Just in case - need to verify these scripts after merging of
- https://github.com/opencv/opencv/pull/20507

Reply:
The script completes successfully, OMZ demo works fine.
Do I need to test some specific scenarios?